plugins/tor/tor_

```diff
@@ -1,28 +1,78 @@
-#!/usr/bin/python2
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+'''
+=head1 NAME
+tor_ 
+
+=head1 DESCRIPTION
+Wildcard plugin that gathers some metrics from the Tor deamon
+https://github.com/daftaupe/munin-plugins-tor
+
+Derived from https://github.com/mweinelt/munin-tor
+
+This plugin requires the stem library : https://stem.torproject.org/
+This plugin requires the GeoIP library : https://www.maxmind.com for the countries plugin
+
+Available plugins :
+tor_bandwidth    # Graph the glabal bandwidth
+tor_connections  # Graph the number of connexions
+tor_countries    # Graph the countries represented our connexions
+tor_dormant      # Graph if tor is dormant or not
+tor_flags        # Graph the different flags of the relay
+tor_routers      # Graph the number of routers seen by the relay
+tor_traffic      # Graph the read/written traffic
+
+=head2 CONFIGURATION
+The default configuration is below
+[tor_*]
+user toranon # or any other user/group that is running tor
+group toranon
+env.torcachefile 'munin_tor_country_stats.json'
+env.torconnectmethod 'port'
+env.torgeoippath "/usr/share/GeoIP/GeoIP.dat"
+env.tormaxcountries 15
+env.torport 9051
+env.torsocket '/var/run/tor/control'
+
+To make it connect through a socket modify this way
+[tor_*]
+user toranon # or any other user/group that is running tor
+group toranon
+env.torcachefile 'munin_tor_country_stats.json'
+env.torconnectmethod 'socket'
+env.torgeoippath "/usr/share/GeoIP/GeoIP.dat"
+env.tormaxcountries 15
+env.torport 9051
+env.torsocket '/var/run/tor/control'
+
+=head1 COPYRIGHT
+MIT License
+
+=head1 AUTHOR
+daftaupe <daftaupe[at]protonmail.com>
+'''
 
 from __future__ import print_function
 import collections
 import json
 import os
+import sys
+
+import GeoIP
```

Comment:
Regarding the external dependencies: please use something along the following lines in order to support the usual "autoconf" approach of munin plugins:
```python
try:
    import GeoIP
    import stem
    import stem.control
    import stem.connection
except ImportError:
    # missing dependencies are reported via "autoconf" - thus failure is acceptable here
    pass
```